### PR TITLE
Config feature

### PR DESCRIPTION
**Pull request by vadymmarkov**

### CODE REVIEW

File: Source/PostCellNode.swift [hunk 1]
```diff
@@ -4,14 +4,13 @@ import AsyncDisplayKit
...
   struct Dimensions {
-    static let padding: CGFloat = 10
     static let dividerHeight: CGFloat = 1
   }
 
   let width: CGFloat
 
   var contentWidth: CGFloat {
-    return width - 2.0 * Dimensions.padding
+    return width - 2.0 * Config.Wall.padding
```
Comment (zenangst) [inline Source/PostCellNode.swift]: Why minus `2.0`?

File: Source/Config.swift [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      ]
+
+      public static var views = [
+        NSFontAttributeName: UIFont.italicSystemFontOfSize(12),
+        NSForegroundColorAttributeName: UIColor.grayColor()
+      ]
+    }
+  }
+
+  public struct Detail {
```
Comment (zenangst) [inline Source/Config.swift]: Is this needed?

**Comment by vadymmarkov** [comment]
@zenangst To subtract left and right padding.

**Comment by zenangst** [comment]
@vadymmarkov Oh, I'm an idiot, don't mind me! :scream:

File: Source/Config.swift (hunk 2 above)
Comment (vadymmarkov) [inline Source/Config.swift]: Maybe we'll need it later, but probably could remove it now.

Comment (zenangst) [inline Source/Config.swift]: Remove it from this pull request if it's not needed meow :scream_cat:

Comment (vadymmarkov) [inline Source/Config.swift]: Done.

**Comment by zenangst** [comment]
:tada:

**Pull request merged**